stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -280,11 +306,22 @@ def transform(self, obj):
                 self._transform(object_map, observation, ds_map, ds_key, obj)
         else:
             self.logger.debug("Not a dict: {}".format(obj))
-        
+
+        # Add unmapped attributes and required property to the observation if it wasn't added via the mapping
+        if self.options.get('unmapped_fallback'):
+            if self.cust_attributes:
+                self.logger.info('Unmapped fallback is enabled. Adding custom attributes to the obseravble object: {}'.format(self.cust_attributes)) 
+                observation.update({"x_" + self.data_source: self.cust_attributes})
```

Comment:
Since this will be a custom cybox object, should be `x- + self.data_source` and have the data source and custom attributes in lowercase.

Reply:
done